SConstruct

```diff
@@ -748,3 +752,16 @@ if "env" in locals():
     # TODO: replace this with `env.Dump(format="json")`
     # once we start requiring SCons 4.0 as min version.
     methods.dump(env)
+
+
+def print_elapsed_time():
+    elapsed_time_sec = time.time() - time_at_start
+    if elapsed_time_sec > 60:
+        print(
+            f"[Time elapsed: {int(elapsed_time_sec / 60)} minute{'s' if elapsed_time_sec else ''} and {(elapsed_time_sec % 60):.1f} seconds.]"
```

Comment:
```suggestion
            f"[Time elapsed: {int(elapsed_time_sec / 60)} minute{'' if 60 <= elapsed_time_sec < 120 else 's'} and {(elapsed_time_sec % 60):.1f} seconds.]"
```

`elapsed_time_sec  = 59` -> `0 minutes 59.0 seconds`
`elapsed_time_sec  = 60` -> `1 minute 1.0 seconds`
`elapsed_time_sec  = 120` -> `2 minutes 0.0 seconds`

`if elapsed_time_sec // 60 == 1` could be switched for `if 60 <= elapsed_time_sec < 120`
I tried both by using the below code in Python 3:

```
import timeit; timeit.timeit("elapsed_time_sec = 61; 60 <= elapsed_time_sec < 120", number=10000000); timeit.timeit("elapsed_time_sec = 61; elapsed_time_sec // 60 == 1", number=10000000)
```
with a range of different values instead of the 61. I found that the second option calculated small whole numbers a lot quicker, but the first one calculated both larger numbers and floats faster. For this reason, I chose to use the first option. If any better alternatives are available, please let me know!

Reply:
Thank you, that part actually contains a really sneaky and stupid typo I missed when I had to retype the code (because I messed up a git stash earlier with a formatter) ;)
`'s' if elapsed_time_sec else ''` should be `'s' if elapsed_time_sec > 120 else ''` 
Also, interesting speed comparison, but I don't think speed is something of concern here since it is only executed once.

Reply:
Ah yes, and my `60 <= elapsed_time_sec` wasn't needed because of the preceeding if statement!
Shouldn't it be `'' if elapsed_time_sec < 120 else 's'` as if the _elapsed_time_sec_ was exactly 120 it would be 2 minute**s**?

Reply:
I think the function should be

```python3
def print_elapsed_time():
    elapsed_time_sec = time.time() - time_at_start
    if elapsed_time_sec < 60:
        print(f"[Time elapsed: {elapsed_time_sec:.1f} seconds.]")
    else:
        print(
            f"[Time elapsed: {elapsed_time_sec // 60} minute{'s' if elapsed_time_sec > 120 else ''} and {(elapsed_time_sec % 60):.1f} seconds.]"
        )
```

for the logially implied `<= 60`.

Reply:
Yes, this is better. Changed.